src/explore-education-statistics-admin/src/pages/publication/components/PublicationContactForm.tsx

```diff
@@ -36,16 +36,17 @@ const PublicationContactForm = ({
       .required('Enter a team email')
       .email('Enter a valid team email'),
     contactName: Yup.string().required('Enter a contact name'),
-    contactTelNo: Yup.string().test({
-      name: 'telephone',
-      message: 'Enter a contact telephone',
-      test: value => {
-        if (!value) {
-          return false;
-        }
-        return true;
-      },
-    }),
+    contactTelNo: Yup.string()
+      .trim()
+      .matches(
+        /^[0-9\s]*$/,
+        'Contact telephone must only contain numeric or whitespace characters',
```

Comment:
I just ran this past Cam to make sure he's happy with it, and he said he's happy for us to add an extra validation that telephone numbers begin with a zero, so I'll add that.

I don't believe a number beginning with a zero ensures that it is a UK number - I believe it's `00` for international calls? - so I'll choose a new attribute and validation message with that in mind.

Reply:
It would be a UK number in the sense that you could dial it in the UK and it would be legitimate. See: https://en.wikipedia.org/wiki/Telephone_numbers_in_the_United_Kingdom

> All mobile numbers, nearly all geographic numbers, and nearly all non-geographic numbers have ten national (significant) digits after the [trunk code](https://en.wikipedia.org/wiki/Trunk_prefix) 0.

Reply:
Should all be sorted.

I've named the attribute `PhoneNumber` - happy to change it if you'd prefer something else.